spacesuit/src/value.rs

```diff
@@ -107,6 +107,10 @@ impl SignedInteger {
             Some(self.0 as u64)
         }
     }
+
+    pub fn to_scalar(self) -> Scalar {
```

Comment:
I don't think we need this method. In the places where we convert SignedInteger to Scalar, the destination type is known and there is no ambiguity. So we can simply do `signed_integer.into()` there.

Reply:
_i'm checking out this branch in order to learn some Rust myself and stop adding nonsensical comments to this PR_